libyul/optimiser/StackCompressor.cpp

```diff
@@ -74,15 +74,15 @@ void eliminateVariables(shared_ptr<Dialect> const& _dialect, ASTNode& _node, siz
 
 }
 
-bool StackCompressor::run(shared_ptr<Dialect> const& _dialect, Block& _ast)
+bool StackCompressor::run(shared_ptr<Dialect> const& _dialect, Block& _ast, bool _optimizeStackAllocation)
 {
 	yulAssert(
 		_ast.statements.size() > 0 && _ast.statements.at(0).type() == typeid(Block),
 		"Need to run the function grouper before the stack compressor."
 	);
-	for (size_t iterations = 0; iterations < 4; iterations++)
+	for (size_t iterations = 0; iterations < 6; iterations++)
```

Comment:
Why the iteration change?

Reply:
4 turned out to be a little low for real-world examples. Also, we have the `return` below, so we can even set it higher.